application-analytics-ui/src/main/resources/Analytics/Code/Macros/VisitsOverview.xml

```diff
@@ -0,0 +1,449 @@
+        <disabled>0</disabled>
+        <name>name</name>
+        <number>2</number>
+        <prettyName>Macro name</prettyName>
+        <size>30</size>
+        <unmodifiable>0</unmodifiable>
+        <classType>com.xpn.xwiki.objects.classes.StringClass</classType>
+      </name>
+      <priority>
+        <disabled>0</disabled>
+        <name>priority</name>
+        <number>11</number>
+        <numberType>integer</numberType>
+        <prettyName>Priority</prettyName>
+        <size>10</size>
+        <unmodifiable>0</unmodifiable>
+        <classType>com.xpn.xwiki.objects.classes.NumberClass</classType>
+      </priority>
+      <supportsInlineMode>
+        <disabled>0</disabled>
+        <displayFormType>select</displayFormType>
+        <displayType>yesno</displayType>
+        <name>supportsInlineMode</name>
+        <number>5</number>
+        <prettyName>Supports inline mode</prettyName>
+        <unmodifiable>0</unmodifiable>
+        <classType>com.xpn.xwiki.objects.classes.BooleanClass</classType>
+      </supportsInlineMode>
+      <visibility>
+        <cache>0</cache>
+        <disabled>0</disabled>
+        <displayType>select</displayType>
+        <freeText>forbidden</freeText>
+        <largeStorage>0</largeStorage>
+        <multiSelect>0</multiSelect>
+        <name>visibility</name>
+        <number>6</number>
+        <prettyName>Macro visibility</prettyName>
+        <relationalStorage>0</relationalStorage>
+        <separator>|</separator>
+        <separators>|</separators>
+        <size>1</size>
+        <unmodifiable>0</unmodifiable>
+        <values>Current User|Current Wiki|Global</values>
+        <classType>com.xpn.xwiki.objects.classes.StaticListClass</classType>
+      </visibility>
+    </class>
+    <property>
+      <async_cached>0</async_cached>
+    </property>
+    <property>
+      <async_context/>
+    </property>
+    <property>
+      <async_enabled>0</async_enabled>
+    </property>
+    <property>
+      <code>{{include reference='Analytics.Code.Macros.VelocityMacros'/}}
+
+{{include reference="Licenses.Code.VelocityMacros"/}}
+
+{{velocity}}
+#macro(executeMacro)
+  #set ($discard =$xwiki.ssx.use('Analytics.Code.Macros.VisitsOverview'))
+  #set ($discard =$xwiki.jsx.use('Analytics.Code.Macros.VisitsSummary'))
+  #analytics_displayMacroTitle('Analytics.Code.Macros.VisitsOverview', 'rendering.macro.visitsOverview.description', 'rendering.macro.visitsOverview.name')
+  
+  #analytics_getMacroId('visitsOverview', $macroId)
+  #analytics_getDate()
+  #set ($options = {
+    "analytics.visitsOverview.visits":'nb_visits',
+    "analytics.visitsOverview.averageVisitDuration":'avg_time_on_site',
+    "analytics.visitsOverview.bounced":'bounce_count',
+    "analytics.visitsOverview.nb_actions":'nb_actions_per_visit',
+    "analytics.visitsOverview.maxActionsInOneVisit":'max_actions'
+  })
+  #analytics_createGraph($macroId, $options, "visitsOverview", $date, "VisitsSummary.get")
+
+#end
+
+
```

Comment:
White space is significant in XWiki syntax. Are you sure these empty lines don't generate useless DIVs?